activerecord/lib/active_record/transactions.rb

```diff
@@ -224,6 +224,41 @@ def transaction(**options, &block)
         end
       end
 
+      class CallbackCountDownLatch # :nodoc:
+        def initialize(count, callback)
+          raise ArgumentError, "count must be at least 2" if count < 2
+          @count = count
+          @callback = callback
+        end
+
+        def call
+          @count -= 1
+          @callback.call if @count.zero?
+        end
+      end
+
+      def after_current_transaction_commit(&block)
```

Comment:
I think this probably wants an `all` or something in its name... and maybe even to live somewhere else? I feel like my expectation of the current name, especially, on a specific model, would be that it [forward to `connection.current_transaction.after_commit`].

Reply:
Good point. It's a leftover from a previous iteration where I didn't have a global "after all transactions are commited" method, but one per DB.

I think both can eventually be useful, but the only one I really care about here is the global one, so it makes more sense on `ActiveRecord` itself, and not on `ActiveRecord::Base`.